test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -345,32 +345,28 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 		})
 
 		It("should validate password enforcement configuration", func() {
-			if eng.ExpandedDefinition.Properties.IsUbuntuDistroForAllNodes() {
```

Comment:
if this diff is hard to read try using https://github.com/Azure/aks-engine/pull/1184/files?w=1